Dockerfile

```diff
@@ -52,8 +52,8 @@ RUN pip install pipenv \
     jscpd \
     markdown-link-check \
     && git clone https://github.com/pyenv/pyenv.git --depth=1 "${PYENV_ROOT}" \
-    && echo 'command -v pyenv >/dev/null || export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.bash_profile \
-    && echo 'eval "$(pyenv init -)"' >> ~/.bash_profile \
+    && echo 'command -v pyenv >/dev/null || export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile \
```

Comment:
Reverting the nit, because I missed that the reversion wasn't committed in the last PR.